### ISSUE TITLE
De-HTAN-ify docs

### ISSUE THREAD

**Issue by milen-sage** [Sage-Bionetworks/HTAN-data-ingress-documentation-draft#6]
This may not be as simple as removing all HTAN text references:

1.  screenshots of the app use HTAN's branding currently 
2. certain direct bucket upload instructions need to be reviewed and updated if needed
3. certain project hierarchy instructions could be extended to cover other hierarchy options (I think that's lower priority though since different forks might have different hierarchies; not sure we want to list all of the possibilities in this upstream)

Looping in @jaeddy and @stacey-w-taylor

**Comment by milen-sage** [comment]
One possible plan for these docs maintenance across projects - just to kick off a conversation for better ideas:

1. de-HTAN-ify this repo
2. push "generic" doc updates in this repo
3. for HTAN, PR from this repo to ncihtan's org [repo](https://github.com/ncihtan/HTAN-Data-Ingress-Docs) to update when ready for generic docs release
4. Make HTAN-specific updates at the ncihtan's repo; if anything is generic, PR back to upstream here (have at least one reviewer on the gh-pages branch for PR merge)

Steps 2-4 would be similar for each project, in case there are multiple projects that would be reusing the docs (e.g. NF, Fred Hutch Accelerator, imCORE, etc. would have their own forks of this repo).

**Comment by xdoan** [comment]
Who is expected to do this?

**Comment by ychae** [comment]
We're looping in Stacey to figure out what might be the best approach to this and how it fits into the larger documentation effort -- I'm hoping that that'll help to determine who can work on it. That's a very long way of saying: not sure yet, but maybe me?

**Comment by jaeddy** [comment]
I think it makes sense to wait on an overall strategy for managing these docs across projects (hopefully based on input from @stacey-w-taylor and the Community Docs group — if that's the right forum). Unless there's an urgent need (in which case we can discuss more), I agree that we can wait on the "who" until we have a better idea what "this" will actually be.

**Comment by stacey-w-taylor** [comment]
Agreed with @jaeddy. I am meeting with @ychae and @milen-sage next week so that I can better understand the context for this doc and the scope of what it would take to make it more generic.  @xdoan, please let me know if you would like to join that meeting, too. I am also working on a larger strategy to manage docs across projects, but this is very much in development at this point as I get familiar with the needs of each group/project.

**Comment by milen-sage** [comment]
Sounds good. Wrt scope - based on various discussions for this particular documentation repo that is already shared with HTAN's community - it looks like there is a set of straightforward changes that will make docs reusable across multiple projects. The required changes will take someone familiar with the user facing application and Jekill about 3-4 hours to make, since a lot of the groundwork has been done. There are a couple of different options on the kinds of changes we decide to do, but they require similar amount of work, within this narrower scope. If people don't have bandwidth to chip in, I can volunteer making the changes in line with the broader documentation strategy, which we can discuss during our meeting. Thanks for organizing that @stacey-w-taylor

**Comment by milen-sage** [comment]
Based on conversation with @stacey-w-taylor we are proceeding with
- short term de-HTAN-ed docs in Jekill
- mid/long term translating these docs in CMS 

Wrt 1) above, I think we'd need to crop existing screenshots; we can also include fewer screenshots and describe some of the more minor actions only in text. That way we'd have to modify less images later on too.

**Comment by ychae** [comment]
Branch has been created here: https://github.com/Sage-Bionetworks/HTAN-Data-Ingress-Docs/tree/develop-generic-docs to start removing HTAN references and making it a bit more generic.

**Comment by milen-sage** [comment]
@ychae which .md files would you prefer to tackle? I can start working on the others and we can sync up when back from vacation.

**Comment by ychae** [comment]
I'm looking through them right now to try and decide if I should modify copy first or attempt to add in some variables that can be specified and changed throughout the entire doc. For example, maybe adding in a variable like `consortia` and setting it to your project name (e.g. HTAN, imCORE, etc) in the `_config.yml` file so that it populates it throughout the entire docs.

**Comment by milen-sage** [comment]
Yes - a template would be a very neat feature for sure!

**Comment by BrunoGrandePhD** [comment]
Is there anything I could help with on this issue?

**Comment by ychae** [comment]
@BrunoGrandePhD I won't ever say no to help! 😁 I was looking into using Liquid tags (or another templating feature if you have any recommendations) to replaces some of the "branding" features of the docs. Would you be able to take that up?

**Comment by milen-sage** [comment]
@ychae do you think the generic docs [here](https://brunograndephd.github.io/HTAN-Data-Ingress-Docs/) can be HTAN-ified using the template and toggle @BrunoGrandePhD [has added](https://github.com/BrunoGrandePhD/HTAN-Data-Ingress-Docs/blob/develop-generic-docs/README.md)? 

If the differences between the current HTAN docs and the generic-based ones are relatively minor, we can review with others from the HTAN DCC. Hopefully we'd be good to go and start using the generic docs for generic changes, PR the changes to the HTAN NCI fork, and add/edit there for minor HTAN-specific material.

**Comment by ychae** [comment]
That sounds like a good workflow! I'll see what the HTAN docs look like using Bruno's template and we can review tomorrow before the HTAN meetings on Wednesday.

**Comment by BrunoGrandePhD** [comment]
A first batch of changes to address this task can be found in PR #13. While I still think the text and structure of the docs could be tweaked, that can be done in a separate PR.
